packages/access-api/src/models/delegations.js

```diff
@@ -162,3 +162,20 @@ export function createDelegationRowUpdate(d) {
     bytes: delegationsToBytes([d]),
   }
 }
+
+/**
+ * @param {Array<number> | Buffer | unknown} sqlValue - value from kysely 'bytes' table - in node it could be a Buffer. In cloudflare it might be an Array
```

Comment:
> value from kysely 'bytes' table

Not sure I understand this comment! Isn't kysely the client? What is a "bytes" table?